frontend/src/components/Button/HOC/withDisable.tsx

```diff
@@ -26,7 +26,8 @@ const withDisable =
       }
     }, [status]);
 
-    const isCompleted = status === COURSE_STATUSES.completed;
+    const isCompleted =

```

Comment:
could you plz add additional request after we get successful status test to automatically have only one completed status